### DISCUSSION TITLE
they removed iboss  due to the hsts exploit , they added  Linewize Connect that blocks stuff now , what can it do and how do I bypass it ??

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1297] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
they removed iboss  due to the hsts exploit , they added  Linewize Connect that blocks stuff now , what can it do and how do I bypass it ??

**Comment by akabutnicer** [comment, marked as answer]
honestly, maybe your tears will break the computer eventually; freeing you from this hindrance

**Reply by ghost** [reply]
:(

**Reply by awsome-webdev** [reply]
> honestly, maybe your tears will break the computer eventually; freeing you from this hindrance

i thought your name suggested you might be nice

**Reply by akabutnicer** [reply]
i tend to joke around, whether my name truly implies that is merely up to the interpreter.

**Reply by awsome-webdev** [reply]
😃

**Reply by py660** [reply]
Unfortunately, my implementation of English is compiled. We'll have to see what the compiler says.

**Reply by awsome-webdev** [reply]
😂

**Reply by akabutnicer** [reply]
hilarious! 🤣

**Comment by schoolexploitkid** [comment]
can i have the id of it?

**Comment by ghost** [comment]
Hey I had linewize too.

**Comment by ghost** [comment]
I can show you some proxies but some MIGHT not work for games.

**Reply by ghost** [reply]
bet

**Comment by ghost** [comment]
You can also do ltmeat flood because I did that.

**Reply by ghost** [reply]
thanks
